frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/element-manager.ts

```diff
@@ -140,6 +130,4 @@ export abstract class ElementManager<
  */
 const featureKeys = {
     value: 'elementValue',
-    // TODO: In the future the type should be saved in the element itself
-    type: 'elementType',
 };
```

Comment:
I think you can now change this back to just a string (YAGNI).

Reply:
:+1: